test/string/cleaning_test.rb

```diff
@@ -30,6 +30,10 @@ class CleaningTest < Minitest::Test
       assert_equal 'IP222', 'IP222'.postcodeize(:db)
       assert_equal 'IP222E', 'IP222E'.postcodeize(:db)
       assert_equal 'HANTS', 'HANTS'.postcodeize(:db)
+      # Log10
+      assert '0.0', '0'.clean(:log10)
+      assert '-10.1', '-10.1'.clean(:log10)
+      assert '1.1', '0.04139268515822508'.clean(:log10)
     end
```

Comment:
These should be `assert_equal` statements, not `assert`.

Is it worth defining behaviour for non-numerical inputs, e.g.
```
assert_equal 'HI', 'HI'.clean(:log10)
```

Reply:
I think the last test above should actually be:
```ruby
      assert_equal '0.04139268515822508', '1.1'.clean(:log10)
```

Reply:
Fixed 'assert_equals'.  Added non-numeric tests.  Fixed float test where the values needed reversing.